instrumentation/afl-compiler-rt.o.c

```diff
@@ -150,6 +146,34 @@ u32 __afl_already_initialized_second;
 
 static int __afl_dummy_fd[2] = {2, 2};
 
+/* Running in persistent mode? */
+
+static inline u8 __afl_is_persistent(void) {
+
+  static u8 is_initialized = 0;
+  static u8 is_persistent = 0;
+
+  if (is_initialized) { return is_persistent; }
```

Comment:
this needs to be `if (likely(is_initialized)) ...`

Reply:
I’m going to change the approach like you suggest below, so this shouldn’t be relevant any more.